README.md

```diff
@@ -14,29 +14,33 @@ This library provides functions to test that a DAML application and its bots are
 
 #### Prerequisites
 
-To be able to develop the library you need:
-* sbt
-
-To use the library in a project:
-* download the library from |PUT LOCATION HERE| as a dependency
+To use the library in a project download the library from Maven Central as a dependency. In a Maven project, the following parts need to be present in pom.xml:
```

Comment:
Add the library as a test scoped dependency to your project...

Reply:
I am using the wording "test scoped dependency" here.